scenarios/remote_signals/README.md

```diff
@@ -0,0 +1,37 @@
+## remote_signals
+
+### Description
+
+This scenario tests the throughput of `remote_signals` operations.
```

Comment:
Please can any custom metrics recorded be documented here -> `remote_signal_round_trip`

Reply:
Done: https://github.com/holochain/wind-tunnel/pull/93/commits/97b20b45b519def837e5b7e5c8e58190330e1cc9